Mathlib/Topology/Compactness/Lindelof.lean

```diff
@@ -696,6 +698,25 @@ instance (priority := 100) SecondCountableTopology.toHereditarilyLindelof
     use t, htc
     exact subset_of_subset_of_eq hcover (id htu.symm)
 
+instance SecondCountableTopology.from_pseudometrizable_Lindelof [PseudoMetrizableSpace X]
```

Comment:
```suggestion
instance SecondCountableTopology.ofPseudoMetrizableSpaceLindelofSpace [PseudoMetrizableSpace X]
```
instance names are just one big block. Also, better follow the exact wording of the assumptions than trying to approximate them, because it's more predictable for users.